src/gtkspellcheck/spellcheck.py

```diff
@@ -205,13 +174,15 @@ def move(self, location):
     def __init__(
         self, view, language="en", prefix="gtkspellchecker", collapse=True, params=None
     ):
+        super().__init__()
         self._view = view
         self.collapse = collapse
-        self._view.connect(
-            "populate-popup", lambda entry, menu: self._extend_menu(menu)
-        )
-        self._view.connect("popup-menu", self._click_move_popup)
-        self._view.connect("button-press-event", self._click_move_button)
+        if Gtk.MAJOR_VERSION < 4:
```

Comment:
This is used quite often. I think, it makes sense to have a global variable `_is_gtk3` set to `Gtk.MAJOR_VERSION < 4` and then use it everywhere (would more descriptive and easier to read IMO).

Reply:
For sure, done. You used lowercase in one comment and uppercase in the other, I've used upper, let me know otherwise.